include/mbedtls/config_psa.h

```diff
@@ -615,6 +631,15 @@ extern "C" {
 #define PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY 1
 #endif /* MBEDTLS_ECP_C */
 
+#if defined(MBEDTLS_DHM_C)
+#define MBEDTLS_PSA_BUILTIN_KEY_TYPE_FFDH_KEY_PAIR 1
+#define PSA_WANT_KEY_TYPE_FFDH_KEY_PAIR 1
+#define MBEDTLS_PSA_BUILTIN_KEY_TYPE_FFDH_PUBLIC_KEY 1
+#define PSA_WANT_KEY_TYPE_FFDH_PUBLIC_KEY 1
```

Comment:
If so - it would be great to add a config that would test this configuration!

Reply:
I will add:
```
#if defined(PSA_WANT_KEY_TYPE_FFDH_PUBLIC_KEY)
#if !defined(MBEDTLS_PSA_ACCEL_KEY_TYPE_FFDH_PUBLIC_KEY)
#define MBEDTLS_PSA_BUILTIN_KEY_TYPE_FFDH_PUBLIC_KEY 1
#define MBEDTLS_DHM_C
#define MBEDTLS_BIGNUM_C
#endif /* !MBEDTLS_PSA_ACCEL_KEY_TYPE_FFDH_PUBLIC_KEY */
#endif /* PSA_WANT_KEY_TYPE_FFDH_PUBLIC_KEY */
```
Then it will be consistent with ECC. Anyway in code there is always:
```
#if defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_FFDH_KEY_PAIR) || \
    defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_FFDH_PUBLIC_KEY)
```

I don't understand why key_pair and public_key are separated. We keep only private key and public key is always calculated from private key when needed. So how the public key be supported on its own?